src/AutoRest.CSharp/Mgmt/Decorator/UrlToUri.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+using AutoRest.CSharp.Input;
+
+namespace AutoRest.CSharp.Mgmt.Decorator
+{
+    internal static class UrlToUri
+    {
+        private static readonly char LowerCaseI = 'i';
+
+        public static void UpdateSuffix(IEnumerable<Schema> schemas)
```

Comment:
The 'rename-rules' can provide similar function, the different is here only happens on suffix, would love to combine these 2 into 1 in the future @ArcturusZhang